Source/WebKit/UIProcess/API/ios/WKWebViewIOS.mm

```diff
@@ -3079,7 +3088,7 @@ - (UIEdgeInsets)currentlyVisibleContentInsetsWithScale:(CGFloat)scaleFactor obsc
 
 - (BOOL)_shouldDeferGeometryUpdates
 {
-    return _perProcessState.liveResizeParameters || _perProcessState.dynamicViewportUpdateMode != WebKit::DynamicViewportUpdateMode::NotResizing || _perProcessState.isAnimatingFullScreenExit;
+    return _perProcessState.liveResizeParameters || _perProcessState.dynamicViewportUpdateMode != WebKit::DynamicViewportUpdateMode::NotResizing || _perProcessState.isAnimatingFullScreenExit || _perProcessState.firstTransactionIDAfterInsetAdjustment;
```

Comment:
Bundling `firstTransactionIDAfterInsetAdjustment` into `_shouldDeferGeometryUpdates` is problematic on at least one count, since it doesn't actually let us get a content rect update across with the _adjusted_ visible rect